configs/icds-14lakh.yml

```diff
@@ -0,0 +1,563 @@
+          unit_size: 1800
+        - referenced_field: 'case_indices'
+          unit_size: 380
+        - referenced_field: 'case_transactions_total'
+          unit_size: 515
+        - referenced_field: 'ledgers'
+          unit_size: 450
+        - referenced_field: 'ledger_updates'
+          unit_size: 420
+        - referenced_field: 'images_total'
+          unit_size: 370
+        - referenced_field: 'sms_total'  # TODO should be modeled separately from SMS since it's SMS triggers
+          unit_size: 683  # casetimedscheduleinstance
+    process:
+      cores_per_node: 32
+      ram_per_node: 128
+
+  pg_proxy:
+    usage_capacity_per_node: 300000
+    storage_scales_with_nodes: True
+    storage:
+      group: 'VM_other'
+      static_baseline: 100GB
+      add_buffer: False
+      override_estimation_buffer: 0
+    process:
+      cores_per_node: 16
+      ram_per_node: 64
+
+  pg_main:
+    usage_capacity_per_node: 250000
+    storage_scales_with_nodes: True
+    min_nodes: 2
+    storage:
+      group: 'SSD'
+      static_baseline: 50GB  # to account for other static tables
+      data_models:
+        - referenced_field: 'phone_numbers'
+          unit_size: 550
+        - referenced_field: 'users'
+          unit_size: 1600
+        - referenced_field: 'async_indicators'
+          unit_size: 6626
+        - referenced_field: 'sms_models_total'
+          unit_size: 510  # averaged over all models
+    process:
+      cores_per_node: 32
+      ram_per_node: 128
+
+  pg_synclogs:
+    usage_capacity_per_node: 500000
+    storage:
+      group: 'SSD'
+      data_models:
+        - referenced_field: 'synclogs'
+          unit_size: 210000
+    process:
+      cores_per_node: 32
+      ram_per_node: 128
+
+  couchdb:
+    usage_capacity_per_node: 100000
+    min_nodes: 3
+    storage:
+      group: 'SSD'
+      redundancy_factor: 3
+      static_baseline: 20GB  # to account for other databases
+      data_models:
+        - referenced_field: 'users'
+          unit_size: 600000   # disk size / doc count of icds @ 2017-12-13
+    process:
+      cores_per_node: 16
+      ram_per_node: 64
+
+  es_datanode:
+    usage_capacity_per_node: 25000
+    min_nodes: 3
+    storage:
+      group: 'SAS'
+      redundancy_factor: 2
+      data_models:
+        - referenced_field: 'forms_total'
+          unit_size: 5500
+        - referenced_field: 'cases_total'
+          unit_size: 1800
+        - referenced_field: 'case_transactions_total'
+          unit_size: 250
+        - referenced_field: 'ledgers'
+          unit_size: 500
+        - referenced_field: 'sms_total'
+          unit_size: 675
+    process:
+      cores_per_node: 8
+      ram_per_node: 64
+
+  es_master:
+    static_number: 3
+    storage_scales_with_nodes: True
+    storage:
+      group: 'VM_other'
+      static_baseline: 50GB
+      add_buffer: False
+      override_estimation_buffer: 0
+    process:
+      cores_per_node: 4
+      ram_per_node: 8
+
+  kafka:
```

Comment:
There's a comment on some of the other services that the limits aren't well known. Do you think that applies here? What have you found to be the limiting factor?

Reply:
that probably applies to kafka as well but it's much easier to scale up whereas things like RabbitMQ don't scale horizontally so it's not as easy.